assets/src/application/ui/layout/espressoTable/style/data-cell.scss

```diff
@@ -0,0 +1,17 @@
+.ee-rspnsv-table-body {
```

Comment:
the main responsive table stylesheet is huge, this and `row.scss` is a small attempt to make things a bit more maintainable